### PR TITLE
PIC-2781 update tu use a local report variable

### PR DESCRIPTION
**Pull request by rschallapalli**

### CODE REVIEW

File: server/controllers/shared/autoSaveController.ts [hunk 1]
```diff
@@ -2,22 +2,27 @@ import { Request, Response } from 'express'
...
 interface IPageField {
   id: string
   value: string
 }
 
 export default class AutoSaveController extends SharedController {
-  protected override updateFields = async (pageFields: Array<IPageField>) => {
+  protected override updateFields = async (
+    pageFields: Array<IPageField>,
+    overrideBoolean: boolean,
```
Comment (rschallapalli) [inline server/controllers/shared/autoSaveController.ts:15]: Had to add this to workaround the override

File: server/controllers/shared/sharedController.ts [hunk 2]
```diff
@@ -260,7 +260,7 @@ export default class SharedController {
           ...this.correctFormData(req),
         }
       }
-      if (this.checkFieldValueVersions(req)) {
+      if (this.checkFieldValueVersions(req, this.report)) {
```
Comment (rschallapalli) [inline server/controllers/shared/sharedController.ts:263]: had to pass this.report to respect the change in signature

File: server/controllers/shared/autoSaveController.ts (hunk 1 above)
Comment (theDustRoom) [reply server/controllers/shared/autoSaveController.ts:15]: `overrideBoolean` isn't used as far as I can see

**Review by theDustRoom — CHANGES_REQUESTED** [review]

File: server/controllers/shared/autoSaveController.ts (hunk 1 above)
Comment (rschallapalli) [reply server/controllers/shared/autoSaveController.ts:15]: Yes, but if I omit it there is an error as I cannot exclude it from the method override

Comment (theDustRoom) [reply server/controllers/shared/autoSaveController.ts:15]: Sorry, not paying enough attention.

**Review by theDustRoom — APPROVED** [review]
LGTM, no idea if it will fix the issue but...

**Pull request merged**